docs/modules/ROOT/pages/cli.adoc

```diff
@@ -0,0 +1,99 @@
+= Hazelcast Cloud CLI
+
+Hazelcast Cloud CLI is known as `hzcloud` is a command line tool to make actions on Hazelcast Cloud easily. Hazelcast Cloud offers the leading in-memory computing platform, Hazelcast IMDG, as a fully managed service that integrates with your existing virtual private cloud.
```

Comment:
I don't think we need to describe Hazelcast Viridian here. This page is about the CLI. We also shouldn't be writing about IMDG here.